src/TesterInternal/App.config

```diff
@@ -7,5 +7,11 @@
     <ThrowUnobservedTaskExceptions enabled="false"/>
     <gcServer enabled="true"/>
     <gcConcurrent enabled="false"/>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
Loosely.  When testing, I found that some of our current azure tests, even without my code changes, were failing because they were missing this redirect.  Adding this redirect fixed these tests, and allowed me to vet this code change.